tool/src/org/antlr/v4/codegen/target/KotlinTarget.java

```diff
@@ -84,7 +84,7 @@ public Map<Character, String> getTargetCharValueEscape() {
 	public String getTokenTypeAsTargetLabel(Grammar g, int ttype) {
 		// All tokens are namespaced inside a Tokens object.
 		// Here we simply force the qualification
-		return "Tokens." + super.getTokenTypeAsTargetLabel(g, ttype);
+		return super.getTokenTypeAsTargetLabel(g, ttype);
```

Comment:
Maybe remove this method altogether ?

Reply:
Sure, I've already removed it in my local branch. Currently I'm figuring out if I can fix two remaining failing tests related to Unicode stuff.